### PR TITLE
Updated relevant functions & tests to display timeseries for DataSubset objects.

### PR DESCRIPTION
**Pull request by tomw66**

### CODE REVIEW

File: src/clean_air/visualise/dataset_renderer.py [hunk 1]
```diff
@@ -86,37 +86,37 @@ def render(self):
         elif self.x_coord is None and self.y_coord is None and \
                 self.t_coord is not None:
             self.img_type = 'timeseries'
-            fig = render_plot.Plot(self.plot_list).render_timeseries()
+            fig, ax = render_plot.Plot(self.plot_list).render_timeseries()
```
Comment (crbunney) [inline src/clean_air/visualise/dataset_renderer.py:89]: I don't know what `ax` means here, could you make the variable name more descriptive please?

File: src/clean_air/visualise/dataset_renderer.py [hunk 2]
```diff
@@ -86,37 +86,37 @@ def render(self):
...
 
 
 class TimeSeries:
     """This class should handle inputs and outputs, hopefully.
 
         Args:
-        * lat: latitude coordinate for point of interest (if required)
-        * lon: longitude coordinate for point of interest (if required)
+        * x: x coordinate for point of interest (if required)
+        * y: y coordinate for point of interest (if required)
```
[the remark this answers is not in the document]
Comment (tomw66) [reply src/clean_air/visualise/dataset_renderer.py:105]: Just to make it a bit more generic, as the x/y dimensions won't always be lat/lon (as is the case with this data).

File: src/clean_air/visualise/dataset_renderer.py (hunk 1 above)
Comment (tomw66) [reply src/clean_air/visualise/dataset_renderer.py:89]: Added to allow plot styling through the notebooks, as in the `matplotlib` usage `fig, ax = plt.subplots()
`. I'll change `ax` to `axes` 👍

File: src/clean_air/visualise/dataset_renderer.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (crbunney) [reply src/clean_air/visualise/dataset_renderer.py:105]: Thanks, I asked Barnaby about his answer and clarified that the x and y dimensions are metres on an OSGB projection for the data you're using, which is interesting to be aware of. All makes sense when you stop assuming we'll always be using lat/lon

**Review by crbunney — APPROVED** [review]

**Pull request merged**